docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md

```diff
@@ -29,7 +29,7 @@ dotnet new console
 
 Applications that target .NET 5 and later already have the necessary distributed tracing APIs included. For apps targeting older
 .NET versions, add the [System.Diagnostics.DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/)
-version 5 or greater.
+version 5 or greater. For libraries targetting netstandard, we recommend referencing the latest version of the NuGet package.
 
```

Comment:
I would like to add some disclaimer sentence telling, 

Here’s a clearer version:

If your library targets `netstandard` and references a recent version of `System.Diagnostics.DiagnosticSource`, users consuming the `netstandard` package may see a warning if they’re targeting an unsupported .NET version. Users may need to upgrade to a supported .NET version to avoid issues.

Reply:
Do you still think this is needed now that the guidance is use the oldest? The primary purpose of the doc is teaching people how to use the instrumentation APIs so it feels like this disclaimer is getting somewhat far from the main topic.

Reply:
I am fine not adding it for now.